maro/simulator/scenarios/supply_chain/units/storage.py

```diff
@@ -327,3 +327,53 @@ def get_unit_info(self) -> StorageUnitInfo:
             **super(StorageUnit, self).get_unit_info().__dict__,
             product_list=[i for i in self._product_level.keys()],
         )
+
+
+class SuperStorageUnit(UnitBase):
```

Comment:
If `SuperStorageUnit` provides identical interfaces with `StorageUnit` and there is no inheritance relationship between them, I suggest adding a higher abstraction to define the interfaces. For example, `StorageUnit` => `SimpleStorageUnit` / `SuperStorageUnit`.